develop/schemadoc/Protocol/Protocol.Topologies.Topology.Cell.Exposer.md

```diff
@@ -52,6 +52,7 @@ Also note that, inside the Exposer element, you can add linked tables. In the ex
 >
 >   - If an exposer is defined with a LinkedId, it will receive the same property values. The "System Name" property will be set to the display key of the table defined in Cell@table as resolved by the foreign key relations. If no link can be resolved, then the display key of the original row will be used.
 >   - If a LinkedId element has a columnPid attribute containing a column parameter ID in the LinkedId table, then the alarm's "System Name" property will be set to the value found in the specified column.
+>   - For a correct EPM configuration, from DataMiner 10.4.0 [CU14]/10.5.0 [CU2]/10.5.5 onwards, it is important that the system type for each front-end manager is unique, as it is the system type that will be used to trace EPM objects back to their respective front-end managers. If you for example have an "HFC" and an "IOT" front-end EPM Manager in your system, these cannot both have a "Location" cell. Instead, you can prefix this, e.g. "HFC_Location" and "IOT_Location", respectively.
```

Comment:
front-end manager --> element manager, as frontends and backends must use the same protocol.